docs/week-3/syscalls you didn't know.md

```diff
@@ -0,0 +1,169 @@
+# Syscalls you didn't know
+What power do you really have with your terminal?
+
+We all know about Task manager, "who" never leave us at hard times, where you can observe 
+how powerful is your PC and where you "End task" (kill) processes. 
+
+The question is, can you find more?? And the answer is yes, actually a lot of stuff 
+that are not important but cool to see. `That is the topic for this page: find more`. 
+
+Jokes aside, most of the information provided on this page will strengthen your understanding
+in operating system principles, and it will be easier later to grasp new information.
+
+## vmstat
+
+- Information about block IO and CPU activity in addition to memory.
+
+let's run this in terminal (Linux) that runs vmstat 5 times with a delay of 1 second
+```shell
+vmstat 1 5
+```
+![vmstat picture](/img/vmstat.png)
+
+<details>
+<summary><i>What represents each information ? </i></summary>
+
+- Procs
+    - r: The number of processes waiting for run time.
+    - b: The number of processes in uninterruptible sleep.
+- Memory
+
+    - swpd: the amount of virtual memory used.
+    - free: the amount of idle memory.
+    - buff: the amount of memory used as buffers.
+    - cache: the amount of memory used as cache.
+    - inact: the amount of inactive memory. (-a option)
+    - active: the amount of active memory. (-a option)
+- Swap
+
+    - si: Amount of memory swapped in from disk (/s).
+    - so: Amount of memory swapped to disk (/s).
+- IO
+    - bi: Blocks received from a block device (blocks/s).
+    - bo: Blocks sent to a block device (blocks/s).
+- System
+    - in: The number of interrupts per second, including the clock.
+    - cs: The number of context switches per second.
+- CPU
+  These are percentages of total CPU time.
+    - us: Time spent running non-kernel code. (user time, including nice time)
+    - sy: Time spent running kernel code. (system time)
+    - id: Time spent idle. Prior to Linux 2.5.41, this includes IO-wait time.
+    - wa: Time spent waiting for IO. Prior to Linux 2.5.41, included in idle.
+    - st: Time stolen from a virtual machine. Prior to Linux 2.6.11, unknown.
+    - 

```

Comment:
- Format the Markdown
- Fix buggy CPU display
- Try to enhance relevant keywords, for example:

```
- **System**
    - `in`: The number of interrupts per second, including the clock.
    - `cs`: The number of context switches per second.
```